{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"13 Expected a TypeError but got a RangeError\" }\n    at ../../test262/harness/assert.js:95\n    at ../../test262/test/built-ins/DataView/prototype/getFloat32/detached-buffer-before-outofrange-byteoffset.js:33\n\n","path":"../../test262/test/built-ins/DataView/prototype/getFloat32/detached-buffer-before-outofrange-byteoffset.js","memory_kb":8864,"duration":79924192}